ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type Interface interface {
```

Comment:
Actually, I'm coming back to `ringpop.RingpopObject`. Because that best describes what one is looking for when you see that interface in a type signature.